src/App.scss

```diff
@@ -1 +1,31 @@
-// styles go here
+.App {
+  height: 100vh;
+  background-color: rgb(59, 209, 209);
+  padding: 100px;
+}
+
+.title {
+  text-align: center;
+  font-size: 20px;
+}
+
+.btn-center {
+  display: block !important;
+  width: 50px;
+  margin: 0 auto !important;
+}
```

Comment:
you always can add you own button style without important
name class like "App__removed-button" and add you style